docker/kafka-python/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+docker build -t kafka-python .
```

Comment:
We can build it as the `us.gcr.io` tag to begin with.

Reply:
oh truuu